src/controllers/mentor.controller.ts

```diff
@@ -126,3 +129,34 @@ export const getAllMentorsHandler = async (
     throw err
   }
 }
+
+export const getMenteesByMentor = async (
+  req: Request,
+  res: Response
+): Promise<ApiResponse<Mentee[]>> => {
+  try {
+    const user = req.user as Profile
+    const status: ApplicationStatus | undefined = req.query.status as
+      | ApplicationStatus
+      | undefined
+
+    if (status && !(status?.toUpperCase() in ApplicationStatus)) {
```

Comment:
Here we can either remove the optional chaining operator `?`, because you've already checked if it's truthy.
Or it seems using `if (!(status_?.toUpperCase() in ApplicationStatus)) {` also works.